devel/simde/Portfile

```diff
@@ -6,6 +6,7 @@ PortGroup           compiler_blacklist_versions 1.0
 PortGroup           meson 1.0
 
 github.setup        simd-everywhere simde 0.7.2 v
+revision            1
```

Comment:
Revision bump is not needed if the change is only to fix builds that previously failed, as they will be rerun anyway without the revision bump, so please remove as there is no need to force a rebuild for the OSes that already work.

Reply:
Oh, I didn't know that. I'll remove it.